src/pages/404.tsx

```diff
@@ -3,22 +3,27 @@ import Hero from '../components/Hero';
 import Layout from '../components/Layout';
 
 export default function NotFoundPage(): JSX.Element {
-  const title = 'PAGE NOT FOUND';
+  const title = 'page not found';
   const description = 'You have hit a route that does not exist.';
 
   return (
-    <Layout title={title} description={description}>
+
+    <Layout title={title} description={description} >
+
+      <div style={{textTransform:'uppercase'}} >
       <Hero title={title} />
-      <main style={{ width: '100%' }} className="article-reader">
+      </div> 
+      
+      <main style={{ width:'60%', textAlign :'center', margin:'auto'}} className="article-reader"  >
```

Comment:
Styling for the 404 page should be done in 404.scss. If it doesn't exist you can create it.